src/styles/base/index.scss

```diff
@@ -1,17 +1,14 @@
-* {
-  margin:0; 
-}
-
 html {
   box-sizing: border-box;
 }
 
-header {
+nav {
```

Comment:
it would be better to move this style to your nav module

Reply:
done